ForgeUnity/Assets/BeardedManStudios/Scripts/StandAlone/Objects/Float2.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace BeardedManStudios
+{
+	public partial struct Float2
```

Comment:
A minor nitpick, but it may be interesting to add ToString methods to these new types for easy logging and printing, just like Unity's `Vector3` and friends have.